scripts/signer/signer.ps1

```diff
@@ -0,0 +1,87 @@
+#
+#  Copyright (c) Contributors to the Open 3D Engine Project
+# 
+#  SPDX-License-Identifier: Apache-2.0 OR MIT
+# 
+# 
+
+param (
+    [String[]] $exePath,
+    [String[]] $packagePath,
+    [String[]] $bootstrapPath,
+    [String[]] $certificate
+)
+
+# Get prerequisites, certs, and paths ready
+$tempPath = "C:\\temp"
```

Comment:
Perhaps %TEMP%

Reply:
+1 we define the TEMP folder in the different Jenkin scripts, and we actually define it out of the current dir so it gets stored as part of the EBS mount. e.g. `scripts\build\Platform\Windows\build_windows.cmd`
we may need to also define it in `installer_windows.cmd` script to be available here
Also consider just moving it to `env_windows.cmd` since seems several scripts use it.

Reply:
I can write a quick test for `%TEMP%`, then use `c:\temp` if null. The issue is if this gets run outside of cmake, it may not grab anything from `env_windows.cmd` or `installer_windows.cmd`

Reply:
How about using `[System.IO.Path]::GetTempPath()` to get the temporary directory using powershell?

Reply:
That works too. Note that this is just to ensure that the `engine.exe` bootstrapper (the Wix one, not the o3de one) is extracted and signed. `GetTempFileName` would have been nice, but needs name manipulation to sign and put it back into the bootstrap installer.